### ISSUE TITLE
Cant enter to Marketplace

### ISSUE THREAD

**Issue by Tomi1444** [PrestaShopCorp/ps_mbo#645]
I have ps_mbo 2.33 for 1.7.x
When I try enter to Marketplace, this message shown

![image](https://github.com/PrestaShopCorp/ps_mbo/assets/118079647/ebfa7098-739d-46b1-9d21-88bccdf83b42)

**Comment by intraordinaire** [comment]
Hello @Tomi1444 ,

Thanks for your feedback.

Have you tried to disable/enable the module, or reinstall it ?
It seems that a route has not been correctly created, and it may be a problem with the PS cache that is not well cleared sometime when installing modules.

**Comment by Tomi1444** [comment]
How to reinstall?
Del ps_mbo by ftp, and upload it again?
It is possible to reinstall it without composer? I don't think my server have that such a thing

**Comment by intraordinaire** [comment]
Yes you can do it this way, so you are sure to only have the last versions files...
Or by going to the "Module manager" page, and just uninstall it via the UI, and then relaunch the installation.

**Comment by Tomi1444** [comment]
I'm making a little progress... Now it looks like this:

![image](https://github.com/PrestaShopCorp/ps_mbo/assets/118079647/e552df61-b682-41d3-89fe-6f02b34eb39c)

**Comment by Tomi1444** [comment]
Now when I try to upload again ps_mbo using BO, I have this error

![image](https://github.com/PrestaShopCorp/ps_mbo/assets/118079647/42f73747-b4ab-4f34-8e72-afe82707eaca)


`Type error: Argument 1 passed to PrestaShop\Module\Mbo\UpgradeTracker::postTracking() must be an instance of Module, null given, called in /home/sklepvid/domains/vidos-domofony.pl/public_html/modules/ps_mbo/src/EventSubscriber/ModuleEventSubscriber.php on line 74

[Symfony\Component\Debug\Exception\FatalThrowableError 0]`

**Comment by Tomi1444** [comment]
1) Now I uninstalled EventBuss
2) Installed ps-mbo by BO
3) The same error appeared as at the beginning, when I try enter to "Marketplace"



```
 FatalErrorExceptionError: Class PrestaShopBundle\Form\Admin\Extension\ModifyAllShopsExtension contains 1 abstract method and must therefore be declared abstract or implement the remaining methods (Symfony\Component\Form\FormTypeExtensionInterface::getExtendedType)
--
in ModifyAllShopsExtension.php line 4
```

**Comment by Tomi1444** [comment]
In log i have this:

`[2023-10-10 16:11:04] request.INFO: Matched route "admin_mbo_catalog_module". {"route":"admin_mbo_catalog_module","route_parameters":{"_controller":"mbo.controller.modules:catalogAction","_legacy_controller":"AdminPsMboModule","_legacy_link":"AdminPsMboModule","_route":"admin_mbo_catalog_module"},"request_uri":"https://vidos-domofony.pl/XXXXXXXX/index.php/modules/addons/modules/catalog?_token=XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX","method":"GET"} []
[2023-10-10 16:11:04] security.DEBUG: Read existing security token from the session. {"key":"_security_main","token_class":"Symfony\\Component\\Security\\Core\\Authentication\\Token\\UsernamePasswordToken"} []
[2023-10-10 16:11:04] security.DEBUG: User was reloaded from a user provider. {"provider":"PrestaShopBundle\\Security\\Admin\\EmployeeProvider","username":"XXXXXXXXXXX"} []
[2023-10-10 16:11:04] request.CRITICAL: Uncaught PHP Exception Symfony\Component\Routing\Exception\RouteNotFoundException: "Unable to generate a URL for the named route "admin_mbo_security" as such route does not exist." at /home/sklepvid/domains/vidos-domofony.pl/public_html/var/cache/prod/appProdProjectContainerUrlGenerator.php line 599 {"exception":"[object] (Symfony\\Component\\Routing\\Exception\\RouteNotFoundException(code: 0): Unable to generate a URL for the named route \"admin_mbo_security\" as such route does not exist. at /home/sklepvid/domains/vidos-domofony.pl/public_html/var/cache/prod/appProdProjectContainerUrlGenerator.php:599)"} []
[2023-10-10 16:11:04] security.DEBUG: Stored the security token in the session. {"key":"_security_main"} []`

**Comment by Tomi1444** [comment]
I can't do that. Because then I have the same error.

**Comment by sowbiba** [comment]
Hello. I think you better retry a clean install

Remove the folder by ftp,

Go to DB, table ps_module, find ps_mbo, delete the row, and clear cache

Go to Module manager, upload the MBO zip you previously download from addons or github

**Comment by Tomi1444** [comment]
Not helped :(

**Comment by sowbiba** [comment]
Can you describe what you did ?

**Comment by Tomi1444** [comment]
1.7.8.7
2.3.3 from prestashop addons web page

**Comment by intraordinaire** [comment]
Just in case, what's your PHP version ? And how much RAM is allowed to the PHP process ? (`memory_limit` in your `php.ini`)

**Comment by Tomi1444** [comment]
PHP Version 7.4.33 
1024MB

I wonder, maybe I should find way to enable something from this list?

`--without-gdbm' '--without-pear' '--with-curl=/opt/alt/curlssl11/usr' '--with-openssl-dir=/opt/alt/openssl11' '--with-openssl=/opt/alt/openssl11' '--with-pcre-dir=/opt/alt/pcre2' '--with-kerberos=/opt/alt/krb5/usr' '--enable-litespeed' '--enable-pcntl' '--without-mysqli' '--disable-mbstring' '--disable-bcmath' '--disable-dba' '--disable-dom' '--disable-fileinfo' '--disable-json' '--disable-intl' '--disable-pdo' '--disable-phar' '--disable-posix' '--disable-soap' '--disable-sockets' '--disable-sysvsem' '--disable-sysvshm' '--disable-sysvmsg' '--disable-xmlreader' '--disable-xmlwriter' '--without-zip' '--disable-gd' '--without-imap' '--without-xmlrpc' '--without-xsl' '--without-ldap' '--without-pgsql' '--without-snmp' '--without-sodium' '--without-tidy' '--without-enchant' '--without-pspell' '--without-unixODBC'
`

**Comment by intraordinaire** [comment]
I assume this is fixed [here](https://github.com/PrestaShop/PrestaShop/issues/34236#issuecomment-1766416060).
Glad to see you manage to fix this !
